flams/databases/setup.py

```diff
@@ -95,4 +101,4 @@ def get_blastdb_path_for_modification(modification: str, version=None):
     if not version:
```

Comment:
Change the name to `get_blastdb_name...`

Reply:
You can also pass just modification as a single arg since we store them in a dataclass now